packages/react-native/android/build.gradle

```diff
@@ -74,5 +76,6 @@ dependencies {
     // Therefore we ignore the version of JNA that comes with the Breez Liquid SDK from Jitpack
     // and manually add one that does include the necessary Android platform binaries.
     implementation("com.github.breez:breez-sdk-liquid:${getVersionFromNpmPackage()}") { exclude group:"net.java.dev.jna" } // remove for using locally built bindings during development
+    //implementation "org.jetbrains.kotlinx:atomicfu:0.23.1" 
```

Comment:
Is this commented line intentional?

Reply:
Done [35dadc5](https://github.com/breez/breez-sdk-liquid/pull/927/commits/35dadc57ad78b393c8a3f4e70eeefdc5a86bbf29)